### DISCUSSION TITLE
Change collors/ run scripts from windows command line ?

### DISCUSSION THREAD

**Discussion by ricardodimon** [ixjf/MSIRGB#230] [Q&A]
**Change collors and run scripts from windows command line**

I would like to know if there is any way to apply a color setting or run a script (lua) directly from the Windows Command Prompt.

This necessity is because I'm trying to change the color using _TRIGGERcmd + Amazon Alexa_.

The idea would be for the trigger cmd to execute a command like:
`> MSIRGB.exe --setcolor 255,0,0`
or
`> MSIRGB.exe --runscript "Hue Wheel.lua"`.

Finally, link with Alexa, and plug into routines and voice commands.

**Comment by ixjf** [comment]
I have that planned (see #137) but haven't gotten around to implementing it yet. Don't really know when I will.
